presto-mixin/README.md

```diff
@@ -0,0 +1,129 @@
+# Presto mixin
+
+The Presto mixin is a set of configurable Grafana dashboards and alerts.
+
+The Presto mixin contains the following dashboards:
+
+- Presto overview
+- Presto coordinator overview
+- Presto worker overview
+- Presto logs overview
+
+and the following alerts:
+
+- PrestoHighInsufficientResources
+- PrestoHighTaskFailuresWarning
+- PrestoHighTaskFailuresCritical
+- PrestoHighQueuedTaskCount
+- PrestoHighBlockedNodes
+- PrestoHighFailedQueriesWarning
+- PrestoHighFailedQueriesCritical
+
+## Presto overview
+The Presto overview dashboard provides details on integration status/alerts, workers/coordinators, error failures, data throughput, blocked nodes, and distributed bytes.
+![First screenshot of the Presto cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/presto/screenshots/presto_overview_1.png)
+![Second screenshot of the Presto cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/presto/screenshots/presto_overview_2.png)
+
+## Presto coordinator overview
+The Presto coordinator overview dashboard provides details on various query counts and rates, query execution time, CPU time consumed, CPU input throughput, error failures, JVM metrics, and memory pool information.
+![First screenshot of the Presto coordinator overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/presto/screenshots/presto_coordinator_overview_1.png)
+![Second screenshot of the Presto coordinator overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/presto/screenshots/presto_coordinator_overview_2.png)
+
+## Presto worker overview
+The Presto worker overview dashboard provides details on various task rates, pool sizes, output positions, data throughput, JVM metrics, and memory pool information
+![First screenshot of the Presto coordinator overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/presto/screenshots/presto_worker_overview_1.png)
+![Second screenshot of the Presto coordinator overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/presto/screenshots/presto_worker_overview_2.png)
```

Comment:
Screenshot URLs 404 since these have not yet been uploaded. Let's remove that until we're 100% on the number of screenshots and naming, just to prevent accidents